Assets/Scripts/AgeSelection.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AgeSelection : MonoBehaviour {
+    public Text ageDisplay;
+    private int playerAge;
+    private string input;
+    private int maxAge = 99;
```

Comment:
🤔 do you think someone 100 or older might ever play this because LEGO made this assumption and became a meme https://knowyourmeme.com/photos/1192098-lego